flipman-api/Flipman.Api/Services/PrizesServices.cs

```diff
@@ -0,0 +1,39 @@
+using Flipman.Api.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace Flipman.Api.Services;
+
+public interface IPrizesService
+{
+  public Task<IActionResult> GetPrizes();
+  public Task<IActionResult> GetPrize(int id);
+}
+
+public class PrizesService : IPrizesService
+{
+  private FlipmanDbContext DbContext { get; }
+  public PrizesService(FlipmanDbContext dbContext)
+  {
+    DbContext = dbContext;
+  }
+
+  public async Task<IActionResult> GetPrizes()
+  {
+    var prizes = await DbContext.Prizes.ToArrayAsync();
+
+    return Ok(prizes);
```

Comment:
Nas controllers é possível retornar `Ok(prizes)` pq a controller herda de `ControllerBase`. Nesse caso, ou o service  retorna `prizes` no caminho feliz e joga uma exceção quando for um `BadRequest`, ou retorna  a forma "completa" do `Ok()` que é `return new OkObjectResult(prizes);`, A forma completa do `BadRequest()` é `return new  BadRequestResult();`